.github/workflows/ci.yml

```diff
@@ -76,7 +76,7 @@ jobs:
       nerdctl_version: '2.1.3'
       crictl_version: 'v1.30.0'
       firecracker_version: 'v1.7.0'
-      solo5_version: 'v0.9.0'
+      solo5_version: 'v0.9.1'
```

Comment:
suspect should be the same as below

Reply:
Sorry for that. this change should not have taken place